### DISCUSSION TITLE
where could i download the newest snapshot now ?

### DISCUSSION THREAD

**Discussion by panchengj** [filecoin-project/lotus#9764] [General Q&A] [answered]
in the past ,I can download the snapshot with URL: https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/complete_chain_with_finality_stateroots_latest.car,
now it returns 403 error ,where could i download the newest snapshot now ?

**Comment by rwxr-xr-x** [comment]
https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car

**Reply by panchengj** [reply]
could i download a full node snapshot ?

**Comment by rjan90** [comment, marked as answer]
Hey @panchengj!

The Snapshot service migrated to a Cloudflare R2 backend earlier this month. This will provide globally distributed Filecoin network snapshots. In addition to the migration, the snapshots now provide zstandard compressed snapshots which cuts down the size by almost 50%, from 100GB to 55GB, so downloads will be a lot faster.

You can get the link to snapshots here: https://lotus.filecoin.io/lotus/manage/chain-management/#lightweight-snapshot

**Reply by panchengj** [reply]
Is there a full node snapshot link  ?

**Reply by Reiers** [reply]
Unfortunately, we are no longer offering full node snapshots. However, if you have a specific need, please let us know and we will consider providing one for you.

**Comment by ribasushi** [comment]
> in the past ,I can download the snapshot with URL: https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/complete_chain_with_finality_stateroots_latest.car,

> Is there a full node snapshot link ?

@panchengj the URL you mentioned never contained a full snapshot ( a full snapshot is ~20TiB ). It was exactly the same as the `minimal...` one, except it had all messages ever included in the chain, even those that were *not* executed.

What were you using `complete_chain_with_finality_stateroots_latest.car` for?

**Reply by panchengj** [reply]
for  filecoin  transaction and audit On chain

**Reply by ribasushi** [reply]
This export was **never** useful for that purpose, as it never contained the actual receipts ( you wouldn't know if a message was processed or not ). This is why I stopped producing them in the first place.

There might be systems in the near future ( mid Q1 ) to help with this, but the `.car` export that went away was never a solution to your problem...
